.github/workflows/release-channels.yml

```diff
@@ -0,0 +1,52 @@
+name: ZITADEL Release
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - 'release-channels.yaml'
+  workflow_dispatch:
```

Comment:
I am not sure about this,

IMO it should only run when the files is changed on main and not just on any brunch.
On PR we could run a dry run.

Reply:
I already did some dry runs in the branch, that worked out. 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

![Screenshot 2022-08-04 at 10 19 40](https://user-images.githubusercontent.com/47860090/182799376-b2b18680-12a5-46dd-aa58-908d077e093b.png)

Reply:
Ah nice, did not find that!